includes/class-woocommerce.php

```diff
@@ -22,7 +22,7 @@ final class WooCommerce {
 	 *
 	 * @var string
 	 */
-	public $version = '4.0.0-dev';
+	public $version = '4.0.0-beta.1';
```

Comment:
Why are we updating the version here? I believe that in past releases we only updated the version in the `woocommerce.php` file for betas and RCs. I'm not sure if it is a problem or not, but maybe it is not needed and better not to do it just in case there is a problem that we are not anticipating?

For example, here is the PR that update the version for 3.9.0-beta.1: #25203

Reply:
I actually based it on 3.8 update note from  @vedanshujain: https://github.com/woocommerce/woocommerce/commit/20e1584654174bfd823a806c5d95ef5ad4609a8d

but having another look, it seems like it should read `'4.0.0'`, then?

Reply:
Yes, I think so. This part is a bit confusing to me and maybe we need to better document it.

Reply:
Updated to `'4.0.0'`